src/sentry/static/sentry/app/components/stream/groupChart.jsx

```diff
@@ -6,13 +6,18 @@ import styled from 'react-emotion';
 import BarChart from 'app/components/barChart';
 
 const StyledBarChart = styled(BarChart)`
-  height: 24px;
+  height: ${p => p.height};
```

Comment:
Do we need a styled component here if we're just forwarding the height prop?

Reply:
Nope. I'll clean that up.